general/README.md

```diff
@@ -461,20 +461,26 @@ Constants should be all written in capital letters and in snake_case. (except co
 
 ```js
 //good
-const DEFAULT_VALUE = 'none';
+// ./constants.js
+export const DEFAULT_VALUE = 'none';
+export const TIMEOUT = 3;
+export const config = {};
 
-var TIMEOUT = 3;
+// ./app.js
+import {DEFAULT_VALUE} from "./constants";
 
-var config = {};
+const baseUrl = "/some/url";
+fetch(`${baseUrl}/DEFAULT_VALUE`);
 
 //bad
-const defaultValue = 'none';
-
-var timeout = 3;
-
-var someOtherStaticVariable = 'blah';
+// ./constants.js
+export const defaultValue = 'none';
+export const timeout = 3;
+export const someOtherStaticVariable = 'blah';
 ```
 
+What about all `const` variables? - This is unnecessary, so uppercasing should not be used for constants within a file. It should be used for exported constants however.
```

Comment:
Is this right though? We still like to UPPER_SNAKE_CASE constants within a file that **aren't** exported. For instance:

```js
const TIMEOUT = 3000;

export const wait = (waitTimeout = TIMEOUT) => (
    new Promise((resolve) => setTimeout(resolve, waitTimeout))
)
```

Reply:
Personally, I think it should just be `const timeout = 3000;`. But, I don't have a very strong opinion here...